Purpose: Enable large file support in startpar
Author:  Roger Leigh
Fixes:   Needed on GNU/Hurd
Status:  Not submitted upstream

--- a/startpar/Makefile	2012-02-04 12:15:27.300343104 +0000
+++ b/startpar/Makefile	2012-02-04 12:18:57.363143151 +0000
@@ -20,6 +20,7 @@
 
 CC     = gcc
 CFLAGS = $(RPM_OPT_FLAGS) $(COPTS) -D_GNU_SOURCE $(INC) -Wall -W -pipe
+CFLAGS += $(shell getconf LFS_CFLAGS)
 
 ifeq ($(MAKECMDGOALS),makeboot)
 CFLAGS += -DTEST
